gnmi_server/connection_manager.go

```diff
@@ -0,0 +1,101 @@
+package gnmi
+
+import (
+	"sync"
+	"time"
+	"net"
+	"regexp"
+	log "github.com/golang/glog"
+
+	"github.com/go-redis/redis"
+	sdcfg "github.com/sonic-net/sonic-gnmi/sonic_db_config"
+)
+
+const table = "TELEMETRY_CONNECTIONS"
+
+var rclient *redis.Client
+
+type ConnectionManager struct {
+	connections  map[string]struct{}
+	mu           sync.Mutex
+	threshold    int
+}
+
+func (cm *ConnectionManager) PrepareRedis() {
+        ns := sdcfg.GetDbDefaultNamespace()
+	rclient = redis.NewClient(&redis.Options{
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

Wrong indentation due to mixing tabs/spaces.

Reply:
Fixed.